tests/sentry/digests/backends/test_redis.py

```diff
@@ -107,3 +107,14 @@ def test_missing_record_contents(self):
         # contents were merged back into the digest.
         with backend.digest('timeline', 0) as records:
             assert set(records) == set([record_2])
+
+    def test_large_digest(self):
```

Comment:
Not that I'm aware of, it's probably a good idea though (even if its only used to allow a easy opt-out locally.)